docker/README.md

```diff
@@ -0,0 +1,50 @@
+SELKS on Docker
+===============
+
+Intro
+-----
+
+This version of SELKS is based on docker and intended to provide easier deployment and management.
+
+For informations on standard SELKS implementation, see https://github.com/StamusNetworks/SELKS#selks
+
+Requirements
+------------
+- 2 cores
+- 8 GB of free RAM
+- 10 GB of free disk space (actual disk occupation will mainly depend of the number of rules and the amount of traffic on the network)
```

Comment:
10Gb is good but in a bit more realistic case (SOHO lets say), I am thinking better size disk should is recommended.